lib/src/requests_inspector_widget.dart

```diff
@@ -293,9 +327,11 @@ class _Inspector extends StatelessWidget {
   }
 
   Widget _buildSelectedTab(BuildContext context, {required int selectedTab}) {
+    final inspectorController = context.read<InspectorController>();
+
     return selectedTab == 0
         ? _buildAllRequests(context)
-        : const _RequestDetailsPage();
+        : _RequestDetailsPage(inspectorController: inspectorController);
```

Comment:
You can get the controller from the context inside the widget
so please don't pass it from the outside